crates/proto/src/op/edns.rs

```diff
@@ -55,7 +55,7 @@ impl Default for Edns {
 impl Edns {
     /// Creates a new extended DNS object.
     pub fn new() -> Self {
-        Default::default()
+        Edns::default()
```

Comment:
For situations like these, I'd prefer `Self` (potentially even `clippy::use_self`).

Reply:
ok, I added in the next push.